roles/openshift_metrics/templates/heapster.j2

```diff
@@ -0,0 +1,98 @@
+apiVersion: "v1"
+kind: "ReplicationController"
+metadata:
+  name: heapster
+  labels:
+    metrics-infra: heapster
+    name: heapster
+spec:
+  selector:
+    name: heapster
+  replicas: 0
```

Comment:
This (and the same for the corresponding lines in `heapster.j2` and `hawkular_metrics_rc.j2`) creates a problem when the role is applied again.  When we `oc apply` this file again, the rc's will be scaled to 0 and then back (in `scale.yaml`), restarting the pods, even if nothing changes.